src/common/txsplain.js

```diff
@@ -597,6 +597,36 @@ angular.module('txsplain', [])
             ' offer #<b>' + tx.tx.OfferSequence + '</b>.'
         }
 
+        // renderEscrowCreate
+        function renderEscrowCreate() {
+          var html = ''
+          if (tx.tx.Destination && tx.tx.Destination !== tx.tx.Account) {
+            html = 'The escrow is from <account>' + tx.tx.Account +
+              '</account> to <account>' + tx.tx.Destination + '</account><br/>'
+          }
+          html += 'It escrowed ' +
+            '<amount>' + displayAmount(tx.tx.Amount) + '</amount>'
+          if (tx.tx.Condition)
+            html += '<br/>Condition: ' + tx.tx.Condition
+          if (tx.tx.CancelAfter) {
+            html += '<br/>It can be cancelled after <date>' +
+              moment((RIPPLE_EPOCH + tx.tx.CancelAfter)*1000).format('LLL') +
+              '</date>.'
+          }
+          if (tx.tx.FinishAfter) {
+            html += '<br/>It can be finished after <date>' +
+              moment((RIPPLE_EPOCH + tx.tx.FinishAfter)*1000).format('LLL') +
+              '</date>.'
+          }
+          return html
+        }
+
+        // renderEscrowFinish
+        function renderEscrowFinish() {
```

Comment:
```The escrow cancellation was triggered by rXYZ…
The escrowed amount of <amount> was returned to rXYZ (<amount - tx.fee> after transaction costs)
The escrow was created by transaction  <prevTXhash from escrow node> link to transaction

The escrow completion was triggered by rXYZ…
The escrowed amount of <amount> was delivered to rXYZ (<amount - tx.fee> after transaction costs)
The escrow was created by transaction  <prevTXhash from escrow node> link to transaction```

the delivered/returned amount can be confusing since any account can trigger the cancel of finish transaction, including the destination/originator.  On an escrow cancel when the tx.Account is the originator, I think the parenthesis explaining the balance change is the amount returned - the fee should be included.  if any other account triggers the cancel, don't include it.  Same for escrowFinish when the destination is the triggering account. All the info is in the deleted escrow node.